config/dev.env.js

```diff
@@ -6,17 +6,14 @@ const prodEnv = require('./prod.env');
 module.exports = merge(prodEnv, {
   NODE_ENV: '"development"',
 
-  // 单独提供给addresses页面的接口，golang实现的
-  EXPLORE_URL: '"https://explorer.ont.io/"',
-  TEST_EXPLORE_URL: '"https://polarisexplorer.ont.io/"',
+  EXPLORE_URL: '"http://ec2-3-17-77-100.us-east-2.compute.amazonaws.com:8080/"',
+  TEST_EXPLORE_URL: '"http://ec2-18-189-125-117.us-east-2.compute.amazonaws.com:8080/"',
 
-  // 基础API接口
-  API_URL: '"https://explorer.ont.io/v2"',
-  TEST_API_URL: '"https://polarisexplorer.ont.io/v2"',
+  API_URL: '"http://ec2-3-17-77-100.us-east-2.compute.amazonaws.com:8080/v2"',
+  TEST_API_URL: '"http://ec2-18-189-125-117.us-east-2.compute.amazonaws.com:8080/v2"',
 
-  // 提供节点统计信息计算查询的接口：
-  DAPP_NODE_URL: '"https://dappnode1.ont.io:10334"',
-  TEST_DAPP_NODE_URL: '"https://polaris1.ont.io:10334"',
+  // DAPP_NODE_URL: '"https://dappnode1.ont.io:10334"',
```

Comment:
Please add a description of why this is commented or remove commented lines.

Reply:
fixed